Reverse the polarity of the over_13 ribbon

kitsuta opened the pull request:
To reflect changes in reg workflow, we need to assign the formerly-named 'over 13' ribbon only to attendees UNDER 13. This makes the necessary changes. Fixes https://github.com/magfest/ubersystem/issues/2237.

kitsuta committed:
Reverse the polarity of the over_13 ribbon
To reflect changes in reg workflow, we need to assign the formerly-named 'over 13' ribbon only to attendees UNDER 13. This makes the necessary changes. Fixes https://github.com/magfest/ubersystem/issues/2237.

binary1230 reviewed (approved):
looks fine, 1 question: what will happen to attendees in the database that already have over_13 ribbons assigned (assuming there are any)

will we have to run the resave-all-attendees sep command? or maybe zero out these ribbons?

kitsuta commented:
Oh, hey, good point. We'd want to actually keep over_13 and just add under_13, otherwise it'll probably get mad.

kitsuta committed:
Add over_13 back in for Prime

Since we've been running this instance, and over_13 ribbons exist, we need to add them back or it'll asplode.

kitsuta commented:
But yes, we'll want that resave-all-attendees sep command. No big rush, but it should be run sometime before we open.

binary1230 reviewed (commented):
cool +1
